hack/install-vm.sh

```diff
@@ -47,26 +48,37 @@ AGENT_LOG_DIR="/var/log/antrea"
 AGENT_CONF_PATH="/etc/antrea"
 OVS_BRIDGE="br-int"
 OVS_VSWITCHD="ovs-vswitchd.service"
+OVS_BIN="start_ovs"
 
-# Optional arguments
-VALIDATE_OVS_CONFIG=false
+# Antrea agent files
+ANTREA_AGENT_BIN="antrea-agent"
+ANTREA_AGENT_KUBECONFIG="antrea-agent.kubeconfig"
+ANTREA_AGENT_ANTREA_KUBECONFIG="antrea-agent.antrea.kubeconfig"
+
+# Docker variables
+ANTREA_VERSION="1.9.0"
+ANTREA_DOCKER_IMAGE="projects.registry.vmware.com/antrea/antrea-ubuntu:v${ANTREA_VERSION}"
+DOCKER_COMPOSE_VERSION="1.29.2"
+DOCKER_COMPOSE="https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose"
+DOCKER_COMPOSE_DIR="/usr/local/bin/docker-compose"
 NODE_NAME="$(hostname)"
+OS_NAME=""
 
-# List of supported OS versions, verified by antrea.
```

Comment:
nit: comment got removed, re-add the comment.

Reply:
Done.